src/main/java/test/roman/TestAPIPS.java

```diff
@@ -0,0 +1,161 @@
+    @Test(priority = 2)
+    public void checkCountCustomersAfterAddCorrCust() {
+        Assert.assertTrue((CountCustBeforeAdding + 1) == CountCustAfterAdding);
+        System.out.println("A count of customers is increased. New customer added successfully.\n");
+    }
+
+
+    @Severity(SeverityLevel.CRITICAL)
+    @Description("Check values in DB (firstName, lastName, email, birthDay) of added customer with correct values")
+    @Test(priority = 3)
+    public void checkValuesCustAfterAddCorrectCust() {
+
+        // Get values of customer from XMLFileCust
+        String strValuesCust[] = getCustValuesFromXMLFile(xmlCorrectCust);
+
+        given()
+                .auth()
+                .basic(PSKey, "")
+                .when()
+                .get(PSUrl + "/api/customers/" + strIDCust)
+                .then()
+                .statusCode(200)
+                .assertThat()
+                .body("prestashop.customer.firstname", CoreMatchers.is(strValuesCust[0]))
+                .body("prestashop.customer.lastname", CoreMatchers.is(strValuesCust[1]))
+                .body("prestashop.customer.email", CoreMatchers.is(strValuesCust[2]))
+                .body("prestashop.customer.birthday", CoreMatchers.is(strValuesCust[3]));
+    }
+
+
+    @Severity(SeverityLevel.CRITICAL)
+    @Description("Check API's status code after adding customer with existing value of login (mail)")
+    @Test(priority = 4)
+    public void checkAPIStatusCodeAfterAddWrongLoginCust() {
+        int getStatusCode = xmlPostRequestAddCust(xmlExistingLoginCust);
+        Assert.assertFalse(getStatusCode == 201);
+        System.out.println("API return status code: " + getStatusCode + ". New customer with existing login not added.\n");
+    }
+
+
+    @Severity(SeverityLevel.CRITICAL)
+    @Description("Check count of customers after adding customer with wrong value of login (mail)")
+    @Test(priority = 5)
+    public void checkCountCustomersAfterAddWrongLoginCust() {
+        Assert.assertTrue(CountCustBeforeAdding == CountCustAfterAdding);
+        System.out.println("A count of customers is not increased. New customer with wrong login not added.\n");
+    }
+
+
+    @Severity(SeverityLevel.CRITICAL)
+    @Description("Check API's status code after adding customer with wrong value of password (<5 chars)")
+    @Test(priority = 6)
+    public void checkAPIStatusCodeAfterAddWrongPassCust() {
+        int getStatusCode = xmlPostRequestAddCust(xmlWrongPassCust);
+        Assert.assertFalse(getStatusCode == 201);
+        System.out.println("API return status code: " + getStatusCode + ". New customer with wrong password not added.\n");
+    }
+
+
+    @Severity(SeverityLevel.CRITICAL)
+    @Description("Check count of customers after adding customer with wrong value of password (<5 chars)")
+    @Test(priority = 7)
+    public void checkCountCustomersAfrerAddWrongPassCust() {
+        Assert.assertTrue(CountCustBeforeAdding == CountCustAfterAdding);
```

Comment:
there is no direct action in this test
if test before it (that has action) fails - then this one makes no sense
move this validation step to the test above

Reply:
Done, its fixed. A new request created.